python/nav/mibs/pdu2_mib.py

```diff
@@ -186,3 +186,33 @@ def get_over_current_protection_sensors(self):
                 sensor['unit_of_measurement'] = 'boolean'
             result.append(sensor)
         returnValue(result)
+
+    @defer.inlineCallbacks
+    def get_serial_number(self):
+        """Tries to get a serial number from an Raritan device"""
+        candidates = [k for k in self.nodes.keys()
+                      if 'pduSerialNumber' in k]
```

Comment:
This search for MIB objects containing the word `pduSerialNumber` (and likewise for `pduModel` and `boardFirmwareRevision` further down) seems pointless? The PDU2-MIB (or at least the version I have access to) only contains one of each such object, so it would be more concise to just use 

```python
serial = yield self.get_next('pduSerialNumber')
```

Am I missing something here?